apps/emqx/src/emqx_cm.erl

```diff
@@ -448,20 +448,22 @@ kick_session(Action, ClientId, ChanPid) ->
                           , action => Action
                           , error => Error
                           , reason => Reason
-                          })
+                          },
+                #{clientid => unicode:characters_to_list(ClientId, utf8)})
```

Comment:
We needed this because it was `~p` used to format client ID.
Is it still the case? 

If client ID is formatted specifically with "~ts", then it can be binary ?
If this is for filtering, why not compare two binaries ?

Reply:
Not really.

emqx_logger_textfmt is fallback to logger_formatter.
It's template(in emqx_conf_schema.erl) is use clientid, mean: 

if we use emqx_logger_textfmt, is will format clientid as `~p`

Next step:
I will remove clientid and peername in emqx_logger_textfmt's template, and format it in emqx_logger_textfmt,

Then we can compare two binaries!